src/taxes/components/TaxChannelsMenu/TaxChannelsMenu.tsx

```diff
@@ -0,0 +1,75 @@
+import { Card } from "@material-ui/core";
+import ListItemLink from "@saleor/components/ListItemLink";
+import Skeleton from "@saleor/components/Skeleton";
+import { TaxConfigurationFragment } from "@saleor/graphql";
+import {
+  List,
+  ListHeader,
+  ListItem,
+  ListItemCell,
+  makeStyles
+} from "@saleor/macaw-ui";
+import { taxesMessages } from "@saleor/taxes/messages";
+import { channelsListUrl } from "@saleor/taxes/urls";
+import clsx from "classnames";
+import React from "react";
+import { FormattedMessage } from "react-intl";
+
+interface TaxChannelsMenuProps {
+  channels: Array<TaxConfigurationFragment["channel"]>;
+  selectedChannelId: string;
+}
+
+const useStyles = makeStyles(
+  theme => ({
+    clickable: {
+      cursor: "pointer"
+    },
+    scrollWrapper: {
+      overflowY: "scroll",
+      maxHeight: 600
+    },
+    selected: {
+      borderLeft: `4px solid ${theme.palette.saleor.active[1]}`,
+      "& div": {
+        marginLeft: "-4px"
+      }
```

Comment:
You can use `:before` with `position: absolute` and `overflow: hidden` on container too, a little more verbose but you won't risk misplacing nested elements